### DISCUSSION TITLE
Aprendendo a aprender

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#1125] [Discussão]
Cada um tem uma forma mais produtiva para estudar diversas coisas, muitos conseguem estudar melhor sozinhos em casa (on-line), já outros precisam estar presencialmente cara-a-cara com o professor tirando as dúvida na hora.

> Eu, particularmente, tenho muito problema para focar em uma coisa e ficar naquilo por mais de 1 hora e sofro bastante em conseguir estudar de casa ou presencialmente, pois perco facilmente a atenção (não, nunca procurei acompanhamento médico para isso, não sei se chega a ser um distúrbio meu).

Como vocês fazem para focar nos estudos? Onde se sentem mais confortáveis e mais produtivos para aprender uma tecnologia nova? Finalizando, como aprenderam a estudar? (parece algo bobo, mas realmente tenho MUITA dificuldade)

**Comment by brunover** [comment]
Recentemente saiu um episódio do Nerdologia sobre "Como aprender melhor e mais fácil", é uma propaganda do "Curso intesivo PH" mas toca em pontos bem interessantes. Veja se algumas das técnicas mencionadas te ajudam e use como ponto de partida:

https://www.youtube.com/watch?v=n5U83znAsbI

**Comment by VZiemer** [comment]
Também tenho muito problema pra manter o foco (transtorno de atenção já diagnosticado, mas lidando com isso sem tratamentos), o que me motiva a aprender é fazer na pratica, com um objetivo específico, aprendi Javascript e posteriormente AngularJS executando um projeto que atualmente estou reescrevendo pela terceira ou quarta vez.

**Comment by gfmarcelo** [comment]
Boa tarde galera!

Vou compartilhar um pouco do que passei e talvez lhe ajude em algo. 

Notei que com o tempo e sem muito foco deixei de crescer profissionalmente devido ter postergado o estudo de algo ou não ter me capacitado para executar alguma atividade e chegou uma hora que a vida me cobrou isto e me cobra todo dia! 
Então passei a ler sobre I.E. [não o internet explorer e sim a inteligencia emocional] e vi que a falta de foco pode ser influencia de uma serie de fatores como por exemplo, o ambiente, o seu proposito, ansiedade, problemas externos e por ai vai. 
Após encontrar a razão da minha falta de foco, procurei entender e estudar algumas metodologias para aplicar as minhas metas e objetivos e uma que me ajudou muito foi a leitura do minibook kanban e scrum - obtendo o melhor de ambos, disponível  neste site aqui https://www.infoq.com/br/minibooks/kanban-scrum-minibook.

Enfim, deixo algumas dicas:

- Faça uma lista de tudo que você tem a fazer e elimine tudo que esta pendente ou que possa vir a lhe atrapalhar. 
- Tenha uma conversa franca consigo mesmo, defina o que você quer e trace metas de longo, médio e curto prazo. 
- Procure entender a conquista de cada estágio e em uma regressão de metas de longo prazo para a de menor. 
- Entendendo o porque estudar algo e com as metas definidas, aplique dentro de um framework que melhor agrade e tacalhepau, execute uma a uma e não vai ter desculpa.

Livros que possam vir a lhe ajudar

Aprendendo a Aprender. Como Ter Sucesso em Matemática, Ciências e Qualquer Outra Matéria - Barbara Oakley (Este tem até curso no https://coursera.org)
A arte de fazer acontecer - O método GTD - Getting Things Done -  David Allen
A Única Coisa - o Foco Pode Trazer Resultados Extraordinários Para Sua Vida - Gary Keller e Jay Papasan
Inteligência Emocional - Daniel Goleman
Foco - Daniel Goleman

A maioria deste livros estão comentados no podcast do canal resumocast https://www.resumocast.com.br/

Espero ter ajudado.

Um abraço a todos.

**Comment by didiraja** [comment]
Pra mim, o melhor método sempre foi aprender o suficiente para que pudesse recorrer ao Google enquanto faço projetos. O lance da prática é essencial, só me sinto seguro quando posso pensar em alguma coisa e conseguir pelo menos iniciar o processo.

Sobre as aulas em si, presencial com professor sempre será a melhor. Mas as video-aulas com exercicios tem funcionado bastante. Principalmente, quando tem algum fórum para se tirar duvidas acoplado a plataforma.

**Comment by mahenrique94** [comment]
Bom tópico @kvnol. 

Eu gosto de estudar em casa, de boa, em um ambiente silencioso onde posso me concentrar mais, coloque o fone e faço meus cursos ou alguma leitura (artigos, docs ou livros).

Se o assunto me interessar, gosto de criar posts ou vídeos para ele, dessa maneira consigo praticar a fortalecer o conhecimento.

**Comment by JosielLima** [comment]
Tem um curso no Coursera com esse título.
https://pt.coursera.org/learn/aprender

Fiz ele a uns 3 anos e foi muito útil. Fiz de forma gratuita, sem certificação. 
O melhor que alguns vídeos estão com legendas em português. Recomendo.
